druid.go

```diff
@@ -185,6 +201,12 @@ func WithBasicAuth(username, password string) ClientOption {
 	}
 }
 
+func WithSkipTLSVerify(skip bool) ClientOption {
+	return func(opts *clientOptions) {
+		opts.skipTLSVerify = skip
```

Comment:
probably to be replaced with the content of `InsecureSkipVerify` function

Reply:
OK